partitura/score.py

```diff
@@ -4613,12 +4656,20 @@ def unfold_part_maximal(part, update_ids=True, ignore_leaps=True):
         The unfolded Part
 
     """
+    if isinstance(score, Score):
+        new_score = deepcopy(score)
+        new_partlist = list()
+        for part in new_score.parts:
+            unfolded_part = unfold_part_minimal(part, update_ids=update_ids, ignore_leaps=ignore_leaps)
+            new_partlist.append(unfolded_part)
+        new_score.parts = new_partlist
+        return new_score
 
     paths = get_paths(
-        part, no_repeats=False, all_repeats=True, ignore_leap_info=ignore_leaps
+        score, no_repeats=True, all_repeats=True, ignore_leap_info=ignore_leaps
```

Comment:
`all_repeats` is also superfluous/overridden by `no_repeats`, maybe just remove it from the call